### ISSUE TITLE
Change Housing Database ID primary field to Autonumber

### ISSUE THREAD

**Issue by trevorshannon** [philhawksworth/the-united-effort-orginization#106]
The Housing Database table contains records with an ID number as the primary field.  These ID numbers match the legacy DB ID numbers.  However, because the field is just a simple numerical field, adding a new record means manually typing in the next ID in the sequence.  Airtable has an "Autonumber" field type that will auto-increment for you.  However, applying this field type re-numbers all records and obliterates the previous legacy ID numbers.

This has implications for the website, because the URL for an individual property uses that ID number (e.g. for property 883: theunitedeffort.org/housing/affordable-housing/883). So if we change the ID numbers, URLs will change.  Perhaps not a big deal as the website is just getting off the ground, but it needs to be considered.

Another possibility is to have a secondary "legacy ID" or "display ID" field that gets calculated using a formula.  This might be the best solution, as we can use the "display ID" for website URLs and the true primary field ID as the link between Housing Database and Units.  The primary field would be Autonumber, so auto-incrementing.  The "display ID" would be formula-based so also auto-incrementing.  When adding a new record, no attention would need to be paid to any ID numbers at all.  The one snag is that the legacy ID numbers have some missing values (perhaps some records were deleted at some point) so the main unknown is whether we can come up with a clever formula to recreate existing "display IDs" _and_ automatically provide new "display IDs"

@philhawksworth your thoughts?

**Comment by trevorshannon** [comment]
Ok, actually I came up with a better idea.  I temporarily re-inserted records for all the missing values in the original list of ID numbers.  Then I was able to change the primary field to an Autonumber and set up a new calculated field called `DISPLAY_ID` that will be used for website URLs and such.  `DISPLAY_ID` is a simple offset from the primary field ID of +836.  Then I could re-delete the temporary records I added.

Long story short, the housing ID to use on the website is `DISPLAY_ID` and that field conveniently populates itself when adding a new record to the housing database table. The primary field `ID` is now an Airtable-internal field only.
